src/ansys/fluent/core/services/interceptors.py

```diff
@@ -11,6 +11,20 @@
 
 network_logger = logging.getLogger("pyfluent.networking")
 log_bytes_limit = int(os.getenv("PYFLUENT_GRPC_LOG_BYTES_LIMIT", 1000))
+truncate_len = log_bytes_limit // 5
+
+
+def _truncate_grpc_str(message):
+    message_bytes = message.ByteSize()
+    message_str = str(MessageToDict(message))
+    if not log_bytes_limit or message_bytes < log_bytes_limit:
+        return message_str
+    else:
+        network_logger.debug(
+            f"GRPC_TRACE: message partially hidden, {message_bytes} bytes > "
+            f"{log_bytes_limit} bytes limit. To see the full message, set PYFLUENT_GRPC_LOG_BYTES_LIMIT to 0."
+        )
+        return f"{message_str[:truncate_len]} < ... > {message_str[-truncate_len:]}"
```

Comment:
Added a function to truncate the string if it is above the log size limit (rather than hide the string completely), as previously suggested by @seanpearsonuk